pegasus_app/models.py

```diff
@@ -29,4 +40,5 @@ class ScheduleDayType(models.TextChoices):
     day = models.CharField(max_length=32, choices=ScheduleDayType.choices)
     open_time = models.TimeField(auto_now=False, auto_now_add=False)
     close_time = models.TimeField(auto_now=False, auto_now_add=False)
-    phone = models.ForeignKey('PhoneNumberCheckConfig', on_delete=models.CASCADE)
+
+    phone_config = models.ForeignKey('PhoneNumberCheckConfig', on_delete=models.CASCADE, related_name='schedules')
```

Comment:
model name and field name ideally match, if they does not, that could possibly mean we need rename field or a model.